modules/mc/hm.nix

```
@@ -0,0 +1,161 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  mcPackage = pkgs.mc;
+  colors = config.lib.stylix.colors.withHashtag;
+in
+{
+  config = mkIf (builtins.elem mcPackage config.home.packages) {
+    home.file.".config/mc/ini".text = ''
+      [Midnight-Commander]
+      skin=stylix
+    '';
+
+    home.file.".local/share/mc/skins/stylix.ini".text = ''
+      [skin]
+      description = Stylix Theme Matching lsd Output
```

If we are removing non-required declarations, it might be better to generalize the description:

```suggestion
      description = Stylix Generated Theme
```